lib/pay/env.rb

```diff
@@ -9,22 +9,18 @@ module Env
     # the current environment.
     #
     # 1. Check environment variable
-    # 2. Check environment scoped credentials, then secrets
-    # 3. Check unscoped credentials, then secrets
+    # 2. Check environment scoped credentials
+    # 3. Check unscoped credentials
     #
     # For example, find_value_by_name("stripe", "private_key") will check the following in order until it finds a value:
     #
     #   ENV["STRIPE_PRIVATE_KEY"]
     #   Rails.application.credentials.dig(:production, :stripe, :private_key)
-    #   Rails.application.secrets.dig(:production, :stripe, :private_key)
     #   Rails.application.credentials.dig(:stripe, :private_key)
-    #   Rails.application.secrets.dig(:stripe, :private_key)
     def find_value_by_name(scope, name)
       ENV["#{scope.upcase}_#{name.upcase}"] ||
         credentials&.dig(env, scope, name) ||
-        secrets&.dig(env, scope, name) ||
-        credentials&.dig(scope, name) ||
-        secrets&.dig(scope, name)
```

Comment:
We need to keep these for backwards compatibility. Instead of deleting them, we can check if we're running Rails 7.2 higher and skip secrets.

Reply:
@excid3 : who do you like this to access and compare Rails version?
```
   Rails::version.gsub(".", "").to_i
```

that will turn "7.0.6" to 706 (integer) and I can do integer comparison.
I can also do:
```
  Rails::verstion.to_f
```

But, that transform "7.0.6" in 7.0 (float).
And it will transform "7.1.***" to 7.1 which can I can do the comparison.

Or do you have any other suggestion?